### PR TITLE
Run perltidy on everything.

### PR DESCRIPTION
**Pull request by zorkian**
All will be well!

### CODE REVIEW

File: bin/bootstrap.pl [hunk 1]
```diff
@@ -19,23 +19,22 @@
 
 $GITHUB_USER //= $ENV{GITHUB_USER} if exists $ENV{GITHUB_USER};
 
-die "Can't find your github username! " .
-    "Try bootstrap.pl --github-user <username>\n"
+die "Can't find your github username! " . "Try bootstrap.pl --github-user <username>\n"
```
Comment (alierak) [inline bin/bootstrap.pl]: Useless use of string concatenation now that the linebreak is gone

File: bin/bootstrap.pl [hunk 2]
```diff
@@ -47,26 +46,26 @@
...
-    configure_dw_upstream( 'dw-free' );
+    configure_dw_upstream('dw-free');
 }
 
 # now get dw-nonfree if it's not there *and* the user has asked for it
 if ( -d "$LJHOME/ext/dw-nonfree/.git" ) {
     say "Looks like you already have dw-nonfree checked out; skipping.";
 }
-elsif ( $DW_NONFREE ) {
+elsif ($DW_NONFREE) {
```
Comment (alierak) [inline bin/bootstrap.pl]: Wait, why is it removing spaces in parens?

File: bin/delete-bogus-media.pl [hunk 3]
```diff
@@ -14,64 +14,70 @@ sub usage {
...
+my $u = LJ::load_user($user)
     or usage();
 
 # Select row from media_versions
-my @mv = $u->selectrow_array( "SELECT mediaid FROM media_versions" .
-                              " WHERE userid=? AND versionid=?", undef,
-                              $u->id, $versionid );
-unless ( @mv ) {
+my @mv =
+    $u->selectrow_array( "SELECT mediaid FROM media_versions" . " WHERE userid=? AND versionid=?",
```
Comment (alierak) [inline bin/delete-bogus-media.pl]: Useless string concatenation now that the linebreak is gone

File: bin/bootstrap.pl (hunk 2 above)
Comment (nfagerlund) [reply bin/bootstrap.pl]: I noticed it seemed to be doing that for method calls but not basic function calls. Maybe it's also omitting them for small runs of parenthesized text? Seems inconsistent.

File: bin/upgrading/import-includes.pl [hunk 4]
```diff
@@ -40,23 +39,30 @@ BEGIN
...
 
     # now get filetime
-    my $ftimedisk = (stat($path))[9];
-    my $ftimedb = $dbh->selectrow_array("SELECT updatetime
-                    FROM includetext WHERE incname=?", undef, $file)+0;
-    
+    my $ftimedisk = ( stat($path) )[9];
+    my $ftimedb   = $dbh->selectrow_array(
+        "SELECT updatetime
+                    FROM includetext WHERE incname=?", undef, $file
```
Comment (alierak) [inline bin/upgrading/import-includes.pl]: The linebreak and indenting inside this string doesn't make sense now that the beginning of the string has been outdented

File: bin/upgrading/update-db-general.pl [hunk 5]
```diff
@@ -18,7 +18,7 @@
 
 mark_clustered(@LJ::USER_TABLES);
 
-register_tablecreate("vgift_ids", <<'EOC');
+register_tablecreate( "vgift_ids", <<'EOC');
```
Comment (alierak) [inline bin/upgrading/update-db-general.pl]: Lolwhut, now it has a space after the open paren but not a space before the close paren

File: bin/upgrading/update-db-general.pl [hunk 6]
```diff
@@ -478,11 +478,11 @@
 )
 EOC
 
-post_create("supportpointsum",
-            "sqltry" => "INSERT IGNORE INTO supportpointsum (userid, totpoints, lastupdate) " .
-            "SELECT userid, SUM(points), 0 FROM supportpoints GROUP BY userid");
+post_create( "supportpointsum",
+          "sqltry" => "INSERT IGNORE INTO supportpointsum (userid, totpoints, lastupdate) "
+        . "SELECT userid, SUM(points), 0 FROM supportpoints GROUP BY userid" );
```
Comment (alierak) [inline bin/upgrading/update-db-general.pl]: Okay, I'm not sure if this is misleading or I just don't like it, but this sort of makes it look like the "SELECT" string is going to be appended to "sqltry" at first glance.

File: bin/upgrading/update-db-general.pl [hunk 7]
```diff
@@ -3094,639 +3104,656 @@
...
-        try_sql("UPDATE user SET caps=8|2    WHERE paidfeatures='paid'");
-        try_sql("UPDATE user SET caps=4|2    WHERE paidfeatures='early'");
-        try_sql("UPDATE user SET caps=2      WHERE paidfeatures='off'");
-    }
+        # upgrade people to the new capabilities system.  if they're
+        # using the paidfeatures column already, we'll assign them
+        # the same capability bits that ljcom will be using.
+        if ( table_relevant("user") && column_type( "user", "caps" ) eq "" ) {
+            do_alter( "user",
+                "ALTER TABLE user ADD " . "caps SMALLINT UNSIGNED NOT NULL DEFAULT 0 AFTER user" );
```
Comment (alierak) [inline bin/upgrading/update-db-general.pl]: Useless string concatenation

File: bin/upgrading/update-db-general.pl [hunk 8]
```diff
@@ -3094,639 +3104,656 @@
...
+            do_alter( "counter", "ALTER TABLE counter MODIFY max INT UNSIGNED NOT NULL DEFAULT 0" );
+        }
 
-    if (column_type("userpic2", "url") eq '') {
-        do_alter("userpic2", "ALTER TABLE userpic2 " .
-                 "ADD url VARCHAR(255) default NULL AFTER location");
-    }
+        if ( column_type( "userpic2", "url" ) eq '' ) {
+            do_alter( "userpic2",
+                "ALTER TABLE userpic2 " . "ADD url VARCHAR(255) default NULL AFTER location" );
```
Comment (alierak) [inline bin/upgrading/update-db-general.pl]: Useless string contatenation

File: bin/upgrading/update-db-general.pl [hunk 9]
```diff
@@ -3094,639 +3104,656 @@
...
-    if (column_type("supportcat", "user_closeable") eq "") {
-        do_alter("supportcat",
-                 "ALTER TABLE supportcat ADD " .
-                 "user_closeable ENUM('1', '0') NOT NULL DEFAULT '1' " .
-                 "AFTER hide_helpers");
-    }
+        # add a status column to polls
+        unless ( column_type( "poll2", "status" ) ) {
+            do_alter( "poll2",
+                "ALTER TABLE poll2 ADD status CHAR(1) AFTER name, " . "ADD INDEX (status)" );
```
Comment (alierak) [inline bin/upgrading/update-db-general.pl]: Useless string concatenation

File: bin/upgrading/update-db-general.pl [hunk 10]
```diff
@@ -3094,639 +3104,656 @@
...
 
-    unless ( column_type("logproplist", "ownership") ) {
-        do_alter("logproplist",
-                 "ALTER TABLE logproplist ADD ownership ENUM('system', 'user') ".
-                 "DEFAULT 'user' NOT NULL");
-    }
+        unless ( column_type( "jobstatus", "userid" ) ) {
+            do_alter( "jobstatus",
+                "ALTER TABLE jobstatus " . "ADD userid INT UNSIGNED DEFAULT NULL" )
+                ;    # yes, we allow userid to be NULL - it means no userid checking
```
Comment (alierak) [inline bin/upgrading/update-db-general.pl]: Do we really need to break that line right before the semicolon?

File: bin/bootstrap.pl (hunk 2 above)
Comment (alierak) [reply bin/bootstrap.pl]: I think maybe it cares whether the thing in parens is a single thing. Class::Subclass is multiple things with an operator in between, but @$arrayref is one thing, "string that looks ( more, complicated" is one thing, etc.

File: cgi-bin/Apache/LiveJournal.pm [hunk 11]
```diff
@@ -599,68 +637,91 @@ sub trans {
...
-            my $should_show_page = ( $u && ! $u->is_visible ) ||
-                                   ( $entry && ! $entry->valid ) ||
-                                   ( $remote &&
-                                       ( $remote->can_manage( $u ) || ( $entry && $remote->equals( $poster ) ) )
-                                   );
+            my $should_show_page =
+                   ( $u && !$u->is_visible )
+                || ( $entry && !$entry->valid )
+                || ( $remote
+                && ( $remote->can_manage($u) || ( $entry && $remote->equals($poster) ) ) );
```
Comment (alierak) [inline cgi-bin/Apache/LiveJournal.pm]: Possibly misleading indentation on the last line; looks similar to the previous line, but nesting level has increased and indentation has not

File: cgi-bin/Apache/LiveJournal.pm [hunk 12]
```diff
@@ -851,31 +932,39 @@ sub trans {
...
 
     # user domains
-    if (($LJ::USER_VHOSTS || $LJ::ONLY_USER_VHOSTS) &&
-        $host =~ /^(www\.)?([\w\-]{1,25})\.\Q$LJ::USER_DOMAIN\E$/ &&
-        $2 ne "www" &&
+    if (
+           ( $LJ::USER_VHOSTS || $LJ::ONLY_USER_VHOSTS )
+        && $host =~ /^(www\.)?([\w\-]{1,25})\.\Q$LJ::USER_DOMAIN\E$/
+        && $2 ne "www"
+        &&
```
Comment (alierak) [inline cgi-bin/Apache/LiveJournal.pm]: Maybe move this && down to the line where it's used?

File: cgi-bin/DW/BusinessRules/InviteCodes/DWS.pm [hunk 13]
```diff
@@ -41,27 +41,39 @@ search_class, and adj_invites).
...
+        search_arg => 30
+    },
+
     # Users with no invites left
-    noinvleft       => { search => \&_search_noinvleft }, # No search arg
-    # Users with no invites left and 1 invitee paid/perm/active in last 30 days
-    noinvleft_apinv => { search => \&_search_noinvleft_apinvitee,
-                         search_arg => 30 },
+    noinvleft => { search => \&_search_noinvleft },    # No search arg
+          # Users with no invites left and 1 invitee paid/perm/active in last 30 days
```
Comment (alierak) [inline cgi-bin/DW/BusinessRules/InviteCodes/DWS.pm]: Strange how it indented this comment (which pertains to the line below), but didn't indent the last similar comment at line 53/63. Maybe a blank line would help it understand that the comment goes with the next line.

File: cgi-bin/DW/Controller/Admin/Importer.pm [hunk 14]
```diff
@@ -25,80 +25,88 @@ use DW::Controller::Admin;
...
 );
 
-
 # view overall import history
 DW::Routing->register_string( "/admin/importer/history/index", \&history_controller );
 
-DW::Controller::Admin->register_admin_page( '/', ,
-    path => 'importer/history',
+DW::Controller::Admin->register_admin_page(
+    '/',,
```
Comment (alierak) [inline cgi-bin/DW/Controller/Admin/Importer.pm]: Well, at least this makes it more obvious that there's an extra comma here

**Pull request merged**